### PR TITLE
Create helper function to write the json file

### PR DESCRIPTION
**Pull request by VladimirKarin**
https://vladimirkarin.atlassian.net/browse/FR-15

### CODE REVIEW

File: utils/storage.js [hunk 1]
```diff
@@ -6,20 +6,36 @@ function getUsers() {
...
+    fs.writeFileSync('./data/ideas.json', JSON.stringify(users), 'utf-8');
+}
+
 function getDonations() {
     const jsonData = fs.readFileSync('./data/donations.json', 'utf-8');
     const jsData = JSON.parse(jsonData);
     return jsData;
 }
 
+function setDonations(users) {
```
Comment (fedosejev) [inline utils/storage.js:29]: Should be `donations`, not `users`

File: utils/storage.js [hunk 2]
```diff
@@ -6,20 +6,36 @@ function getUsers() {
...
+    fs.writeFileSync('./data/donations.json', JSON.stringify(users), 'utf-8');
```
Comment (fedosejev) [inline utils/storage.js:30]: Should be `donations`, not `users`

File: utils/storage.js [hunk 3]
```diff
@@ -6,20 +6,36 @@ function getUsers() {
...
+    fs.writeFileSync('./data/users.json', JSON.stringify(users), 'utf-8');
+}
+
 function getIdeas() {
     const jsonData = fs.readFileSync('./data/ideas.json', 'utf-8');
     const jsData = JSON.parse(jsonData);
     return jsData;
 }
 
+function setIdeas(users) {
```
Comment (fedosejev) [inline utils/storage.js:19]: Should be `ideas`, not `users`

File: utils/storage.js [hunk 4]
```diff
@@ -6,20 +6,36 @@ function getUsers() {
...
+    fs.writeFileSync('./data/ideas.json', JSON.stringify(users), 'utf-8');
```
Comment (fedosejev) [inline utils/storage.js:20]: Should be `ideas`, not `users`

File: utils/storage.js (hunk 1 above)
Comment (VladimirKarin) [reply utils/storage.js:29]: function setIdeas(users) {
    fs.writeFileSync('./data/ideas.json', JSON.stringify(users), 'utf-8');
}
Same as here ideas, not users. Right?

**Review by fedosejev — CHANGES_REQUESTED** [review]
`setUsers`, `setIdeas` and `setDonations` functions have the same code, the only difference is the filename. Can you write a more generic function `setData` that writes daa to JSON file and then reuse it in `setUsers`, `setIdeas` and `setDonations` functions?

File: utils/storage.js [hunk 5]
```diff
@@ -1,25 +1,45 @@
 const fs = require('fs');
 
+function setData(data) {
+    fs.writeFileSync(`./data/${data}.json`, JSON.stringify(data), 'utf-8');
```
Comment (fedosejev) [inline utils/storage.js:4]: If `data` is an array, then `./data/${data}.json` won't work

**Review by fedosejev — APPROVED** [review]
👍

**Pull request merged**